bool[5] sensor